src/raster/processing/dem/hillshade.rs

```diff
@@ -172,10 +173,13 @@ pub enum ShadingMode {
     Igor,
 }
 
-impl Display for ShadingMode {
-    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
-        let s = format!("{self:?}");
-        f.write_str(&s.to_lowercase())
+impl ShadingMode {
+    pub(crate) fn to_gdal_option(&self) -> &'static str {
```

Comment:
I think this is better in a couple of ways:

 - doesn't rely on the unstable `Debug` formatting; it's unlikely to ever change for enums, but I've seen code break when it changed for other types like arrays
 - it's about 10x less code in the binary: https://rust.godbolt.org/z/8chM9W51W
 - easier to understand (?)